### DISCUSSION TITLE
Piston is not moving any blocks

### DISCUSSION THREAD

**Discussion by Santhosh8465e** [GriefPrevention/GriefPrevention#1525] [Q&A]
So after installing grief prevention even in claimed lands i cant use piston its working normally but if any blocks placed like an observer it dosent move config has only 2 lines about it and i gave piston movement ignored and nothing works should i unintall it?

**Comment by bigpresh** [comment]
I'd suggest opening a ticket, providing your GP config and the other details requested (versions, etc).

I'd also recommend a look at the [documentation for the `PistonMovement` setting](https://github.com/TechFortress/GriefPrevention/wiki/Setup-and-Configuration#pistonmovement) but it sounds like you're saying you've got `GriefPrevention.PistonMovement` set to IGNORED already - which should skip all of GP's piston movement checks?

Note also that [pistons can't push certain block types](https://minecraft.fandom.com/wiki/Piston#Limitations) - but that's vanilla Minecraft mechanics, so if that was the problem you'd see the same behaviour without GP installed.
